dipy/viz/app.py

```diff
@@ -797,9 +881,13 @@ def horizon(tractograms=None, images=None, pams=None,
         small animal data you may need to use something smaller such
         as 2.0. The threshold is in mm. For this parameter to be active
         ``cluster`` should be enabled.
-    random_colors : bool
-        Given multiple tractograms have been included then each tractogram
-        will be shown with different color
+    random_colors : string
```

Comment:
Doest it means we cannot have both (tracts and roi)?

Reply:
That's the default option. `dipy_horizon --random_colors --roi_images non-binary.nii.gz binary1.nii.gz binary2.nii.gz tract1.trk tract2.trk` will apply a random color to `binary1.nii.gz`, another to `binary2.nii.gz`, another to `tract1.trk` and a different color to `tract2.trk`.

If the random colors are desired only on the tractograms or on the rois, then the respective options should be selected (`tracts` and `rois`).

See usage in the description of the PR. I'm providing an example of the command used to generate each one of the figures.